### ISSUE TITLE
Indexing very slow

### ISSUE THREAD

**Issue by phaldenby72** [stringham/move-ts#14]
I've recently upgraded to VS Code 1.19.1 and Move TS 1.7.0. I hadn't noticed any real delays for indexing in the past, but I'm now seeing that it hasn't completed after 10 minutes. I don't know if it's just slow or if it's hung.

Any suggestions? It's a great extension and I'd really like to get it working again.

**Comment by stringham** [comment]
What operating system are you using?

**Comment by phaldenby72** [comment]
Thanks for the quick reply! I'm on Windows 7.

**Comment by stringham** [comment]
Yesterday I released Move TS 1.7.0 which was supposed to "Improve performance of indexing the workspace" (https://github.com/stringham/move-ts/commit/da386627c841b827a6e1cf121e20d507f876205a).

I think there was an assumption of using a unix based operating system in one of my optimizations. I've just released version 1.7.1. Will you try updating Move TS and let me know if it solved your problem?

Thanks
